### PR TITLE
BookA-Pset_GeoObsBoreholeConstruction

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_GeoObsBoreholeConstruction

## Description: 

Properties for describing the construction of a borehole

| Property | Datatype | Description |
|----------|----------|-------------|
|DrillingDiameter|IfcPositiveLengthMeasure|Drilled Borehole diameter (mm)|
|LiningMaterial|IfcLabel|The material of the lining for borehole wall stabilizing|
|BoreholeFilter|IfcLabel|Existence and type or properties of filter segment installed in the casing of a borehole|
|BoreholeInstallation|IfcLabel|Equipments installed in the borehole|

### CODE REVIEW

File: IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Existence and type or properties of filter segment installed in the casing of a borehole.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md]: Is it just me having understanding issues with this description?

File: IFC4x3/Properties/b/BoreholeInstallation_0o06vNWW901uy$1qJJssW7/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Equipments installed in the borehole.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/b/BoreholeInstallation_0o06vNWW901uy$1qJJssW7/Documentation.md]: Equipment is a noun that knows only singular form.
```suggestion
Equipment installed in the borehole.
```

File: IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml]: Is this really needed as a property? Quantity or extracted from geometry should be quite okay here.

File: IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LiningMaterial_0mAA1gW8z7mevSgdceBIp5" Name="LiningMaterial" UniqueId="3028a06a-808f-47c2-8e5c-aa79a82d2cc5" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml]: Propose using `IfcPropertyReferenceValue.PropertyReference` : `IfcMaterialDefinition`

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md]: No. Pinging @j-weil

File: IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml]: What if the representation is just a line? Ping @j-weil.

File: IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml (hunk 4 above)
Comment (larswik) [reply IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml]: Fixed in commit https://github.com/bSI-InfraRoom/IFC-Specification/pull/581/commits/f23b2d1e012d12e895e2730c791db8e953e79c53

File: IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml:1]: Should this be `IfcPropertyReferenceValue`.`PropertyReference`=`IfcMaterial`?

**Review by SergejMuhic — COMMENTED** [review]
Just one question.

File: IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml (hunk 3 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml]: Understood. I would like to keep this in mind (unless documented somewhere else) that we need a valid representation usage for 'Axis' for `IfcBorehole`. Is it documented or shall I derive an issue from your comment?

File: IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/b/BoreholeFilter_3fE4eMnCT7eveQISamMQlZ/Documentation.md]: I changed to "Identification of filter..."

File: IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml]: We haved added a usage for this.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**